hw-probe.pl

```diff
@@ -14433,22 +14433,21 @@ ()
                 $Name = "ghostbsd";
```

Comment:
Please remove non-UTF chars from the commit message.

Reply:
Weird, I did haven't touch line 14433.

Reply:
Fixed the commit message